index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Portfolio Website</title>
+</head>
+<body >
+    <nav class="header-logo">
+        <a href="#"><img src="images/My Logo.svg" alt="" class="logo"></a>
+        <img src="images/Enabled.png" alt="" class="image-icon"></nav>
+         
+    </nav>
+
+
+
+    <section class="all-body" id="body-cover">
+        <h2 class="title">I'm Mariane Glad to see you!</h2>
+        <p class="intro">
+            I'm a software developer! I can help you build a product , 
+            feature or website Look through some of my work and experience! 
+            If you like what you see and have a project you need coded, don't 
+            hestiate to contact me.
+        </p>
+    
+        <p class="lets-connect">LET'S CONNECT</p>
+            
+        <ul class="social-buttons">
+            <img src="images/Vector.png" class="icons">
+            <img src="images/Vector (1).png" class="icons"><img src="images/Vector (2).png" class="icons">
+            <img src="images/Vector (3).png" class="icons">
+            <img src="images/Vector (4).png" class="icons">
+            <img src="images/Vector (5).png" class="icons">
+        </ul>
```

Comment:
- [OPTIONAL] KIndly make sure the `li` tag is the primary children's tag of the `ul`. Please make sure to wrap each icon into a `li` tag.

- [OPTIONAL] Kindly make sure to wrap each icon into the Anchor `a` tag. Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using the  `href` attribute.